frontend/src/App.vue

```diff
@@ -165,6 +172,12 @@ export default {
     }
   },
   methods: {
+    isNoGenerators() {
+      return (this.currentPrompt &&
+        this.currentPrompt.name === this.messages.select_generator_name &&
```

Comment:
(_.get(this.currentPrompt, "name")  === this.messages.select_generator_name &&
        _.isEmpty(_.get(this.currentPrompt, "questions[0].choices")) );